_specifications/lsp/3.18/language/signatureHelp.md

```diff
@@ -258,11 +277,17 @@ export interface SignatureInformation {
 	/**
 	 * The index of the active parameter.
 	 *
-	 * If provided, this is used in place of `SignatureHelp.activeParameter`.
+	 * If `null`, no parameter of the signature is active (for example a named
+	 * argument that does not match any declared parameters). This is only valid
+	 * since 3.18.0 and if the client specifies the client capability
+	 * `textDocument.signatureHelp.noActiveParameterSupport === true`
+	 *
+	 * If provided (or `null`), this is used in place of
+	 * `SignatureHelp.activeParameter`.
 	 *
 	 * @since 3.16.0
 	 */
```

Comment:
Same as above.

Reply:
I think what we need to do is to support more than one since tag in the meta model.

Reply:
Created https://github.com/microsoft/vscode-languageserver-node/issues/1204.